assets/shaders/hud_drag_select.frag.glsl

```diff
@@ -0,0 +1,10 @@
+#version 330
+
+// Color of the drag rectangle
+uniform vec4 in_col;
+
+layout(location=0) out vec4 out_col;
+
+void main() {
+	out_col = in_col;
```

Comment:
we could draw the selection box as a big quad, where we only color the outline when the frag position is less than the desired outline width.

Reply:
That would be nice... for a future PR ;)